features/step_definitions/navigation_steps.rb

```diff
@@ -1,3 +1,7 @@
 Given(/^I am on the landing page$/) do
   visit root_path
+end
+
+Given(/^I am on the list applications page$/) do
+  visit memberships_path
```

Comment:
Still a step definition concerning MembershipApplication. Should be extracted to relevant file.